user-lock-manager-w-keypad-support.smartapp.groovy

```diff
@@ -682,6 +711,8 @@ private initialize() {
   subscribe(theLocks, "codeReport", codereturn)
   subscribe(theLocks, "lock", codeUsed)
   subscribe(theLocks, "reportAllCodes", pollCodeReport, [filterEvents:false])
+  subscribe(location,"alarmSystemStatus",alarmStatusHandler)
+  subscribe(keypad,"codeEntered",codeEntryHandler)
```

Comment:
I could definitely do that, what would be the best way to determine if the keypad is enabled/set?